builder/builder.go

```diff
@@ -420,6 +484,11 @@ func (b *Builder) InitMix(clearver string, mixver string, all bool, upstreamurl
 
 	b.UpdateRepo(clearver, all)
 
+	// Set up rpms and local dirs
+	if err := b.createRpmDirs(); err != nil {
```

Comment:
This came up on the last PR, too, and the issue is the same in this one.

Technically speaking, build chroots says, `if b.Repodir != "" {  // run m4 telling it about repodir and whatnot }`, so _I think_ creating these directories in and of itself doesn't tell mixer to generate the local yum.conf, it's _setting the variables in the conf_ (L119 above) that does that.

Either way, the reason @rchiossi put this in was to help automate the process of setting up the infrastructure for incorporating local RPMs. That's still a good goal, but I 100% agree that they shouldn't be created always for all users.

My take would be to create a `--local-rpms` flag for `mixer init-mix` that, if passed, adds the above-mentioned variables to the .conf file and creates these directories. Otherwise, they're omitted. Would be pretty easy to implement.